lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -204,6 +206,33 @@ def add_deferred(post, options)
         }
       end
 
+      def add_three_d_secure(post, payment_method, options)
+        three_d_secure = options[:three_d_secure]
+        return unless three_d_secure
+
+        post[:threeDomainSecure] = {}
+        post[:threeDomainSecure][:eci] = three_d_secure[:eci]
+        post[:threeDomainSecure][:specificationVersion] = three_d_secure[:version]
+
+        if payment_method.brand == 'mastercard'
```

Comment:
❓ Question:

Maybe my memory is failing me but Kushki only considers 3DS for Visa and MasterCard right?, it that is the case what should be the behavior with the rest of the supported card brands?

Reply:
after tests... the behaviour is the same as failed request with code k001

Reply:
Mmm maybe should we consider asking what is the expected behavior ?. if we just let the request fail debugging for a merchant and support is going to be a nightmare, maybe we should raise an Argument exception here or return a failure response with the proper message telling that 3DS is not supported for that brand?

Reply:
I have checked the docs in order to find some other way to handle this but I could not find a more proper way so I add the ArgumentError into the implementation... Hope this is a better way to deal with this.
Thank you for the suggestion!